apps/cowswap-frontend/src/modules/ordersTable/pure/ReceiptModal/index.tsx

```diff
@@ -132,9 +132,14 @@ export function ReceiptModal({
       <styledEl.Wrapper>
         <styledEl.Header>
           <styledEl.Title>Order Receipt</styledEl.Title>
-          {showRecreateModal && (
-            <ButtonSecondary variant={'light'} margin={'0 auto 0 10px'} minHeight={'28px'} onClick={showRecreateModal}>
-              Recreate this order
+          {showAlternativeOrderModal && (
+            <ButtonSecondary
+              variant={'light'}
+              margin={'0 auto 0 10px'}
+              minHeight={'28px'}
+              onClick={showAlternativeOrderModal}
+            >
+              {isPending(order) ? 'Edit' : 'Recreate'} this order
```

Comment:
Should we use `alternativeOrderModalContext.isEdit` instead? Just to be coherent